src/cljs/athens/views/filters.cljs

```diff
@@ -1,230 +0,0 @@
-(ns athens.views.filters
-  (:require
-    ["@material-ui/icons/ArrowDownward" :default ArrowDownward]
-    ["@material-ui/icons/Block" :default Block]
-    ["@material-ui/icons/Check" :default Check]
-    ["@material-ui/icons/FilterList" :default FilterList]
-    ["@material-ui/icons/Sort" :default Sort]
-    [athens.style :refer [color OPACITIES]]
-    [athens.views.buttons :refer [button]]
-    [athens.views.textinput :refer [textinput]]
-    [cljsjs.react]
-    [cljsjs.react.dom]
-    #_[re-frame.core :as re-frame :refer [dispatch]]
-    [reagent.core :as r]
-    [stylefy.core :as stylefy :refer [use-style #_use-sub-style]]))
```

Comment:
Filters never made it from devcards into the app, but I'm not sure sure we should delete this file as we may implement some of this UI again: dropdown + search + filter buttons.

Should we create storybook analogy first? Could also save a permalink to this file somewhere, just need to make sure we can find it. @shanberg

Reply:
It’s still in the GitHub history even if we remove it from the latest code. Do we need to hang on to it?

Reply:
See https://github.com/athensresearch/athens/pull/1564#discussion_r704565717